registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MosipVerticleManager.java

```diff
@@ -317,24 +320,28 @@ private void addTagsToMessageDTO(MessageDTO messageDTO) {
 
 	private Map<String, String> getTagsFromPacket(String id) throws ApisResourceAccessException,
 			PacketManagerException, JsonProcessingException, IOException {
-        InfoRequestDto infoRequestDto = new InfoRequestDto(id);
-
-        RequestWrapper<InfoRequestDto> request = new RequestWrapper<>();
-        request.setId(ID);
-        request.setVersion(VERSION);
-        request.setRequesttime(DateUtils.getUTCCurrentDateTime());
-        request.setRequest(infoRequestDto);
-        ResponseWrapper<InfoResponseDto> response = (ResponseWrapper) restApi.postApi(
-			ApiName.PACKETMANAGER_INFO, EMPTY_STRING, EMPTY_STRING, request, ResponseWrapper.class);
-
-        if (response.getErrors() != null && response.getErrors().size() > 0) {
+		TagRequestDto tagRequestDto = new TagRequestDto(id, null);
+		RequestWrapper<TagRequestDto> request = new RequestWrapper<>();
+		request.setId(ID);
+		request.setVersion(VERSION);
+		request.setRequesttime(DateUtils.getUTCCurrentDateTime());
+		request.setRequest(tagRequestDto);
+		ResponseWrapper<TagResponseDto> response = (ResponseWrapper<TagResponseDto>) restApi
+				.postApi(ApiName.PACKETMANAGER_GET_TAGS, "", "",
+						request, ResponseWrapper.class);
+
+		if (response.getErrors() != null && response.getErrors().size() > 0) {
+			logger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
+					id, JsonUtils.javaObjectToJsonString(response));
```

Comment:
OK, But i see 100+ matches for above logging and since we were following same pattern i used same. Lets remove all together? In case session id is being introduced in future "OR" it can be removed from all places at once?